unity-renderer/Assets/DCLPlugins/ECS7/Systems/GltfContainerLoadingStateSystem/GltfContainerLoadingStateSystem.cs

```diff
@@ -11,6 +11,7 @@ public class GltfContainerLoadingStateSystem
     {
         private readonly IInternalECSComponent<InternalGltfContainerLoadingState> gltfContainerLoadingStateComponent;
         private readonly IECSComponentWriter componentWriter;
+        private int timestamp = 0;
```

Comment:
This timestamp should be entity aware. I think you are setting it globally, is that correct?

Reply:
that was because I wasn't storing it at the crdt state, but I'm changing it right now, so timestamp will be at entity level, being `prevComponentStateForEntity.timestamp +1`